src/src/modules/recipe/entities/recipe.entity.ts

```diff
@@ -0,0 +1,43 @@
+import { Unit } from '../dto/create-recipe.dto';
+import {
+  Column,
+  Entity,
+  ManyToOne,
+  OneToMany,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+
+@Entity({ name: 'recipe' })
+export class Recipe {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @Column()
+  description: string;
+
+  @OneToMany(() => Ingredient, (ingredient) => ingredient.recipe, {
+    cascade: true,
+    eager: true,
+  })
+  ingredients: Ingredient[];
+}
+
+@Entity({ name: 'ingredient' })
+export class Ingredient {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @Column()
+  name: string;
+
+  @Column({ type: 'varchar' })
+  unit: Unit;
+
+  @Column({ type: 'integer' })
+  quantity: number;
+
+  @ManyToOne(() => Recipe, (recipe) => recipe.ingredients, {
+    onDelete: 'CASCADE',
```

Comment:
`onDelete: "CASCADE"` used to remove related columns after delete row from primary table